core/src/validator.rs

```diff
@@ -138,17 +143,29 @@ use {
         path::{Path, PathBuf},
         sync::{
             atomic::{AtomicBool, AtomicU64, Ordering},
-            Arc, Mutex, RwLock,
+            Arc, LazyLock, Mutex, RwLock,
         },
         thread::{sleep, Builder, JoinHandle},
         time::{Duration, Instant},
     },
     strum::VariantNames,
     strum_macros::{Display, EnumCount, EnumIter, EnumString, EnumVariantNames, IntoStaticStr},
     thiserror::Error,
-    tokio::runtime::Runtime as TokioRuntime,
+    tokio::runtime::{self, Runtime as TokioRuntime},
 };
 
+static GLOBAL_RUNTIME: LazyLock<TokioRuntime> = LazyLock::new(|| {
```

Comment:
Do we want to define runtime here or, maybe, it would be better to define it validator/src/main.rs and pass to the `Validator::new`?